src/models/Product.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  AllowNull,
+  BelongsTo,
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  PrimaryKey,
+  Table,
+} from 'sequelize-typescript';
+import { Phone } from './Phone';
+
+@Table({
+  tableName: 'products',
+  createdAt: false,
+  updatedAt: false,
+})
+export class Product extends Model {
+  @PrimaryKey
+  @AllowNull(false)
+  @Column({
+    type: DataType.INTEGER,
+  })
+    id: string;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    category: string;
+
+  @ForeignKey(() => Phone)
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    phoneId: string;
+
+  @BelongsTo(() => Phone)
+    phone: Phone | null;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    itemId: string;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    name: string;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.INTEGER,
+  })
+    fullPrice: number;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.INTEGER,
+  })
+    price: number;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    screen: string;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    capacity: string;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    color: string;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    ram: string;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.INTEGER,
+  })
+    year: number;
+
+  @AllowNull(false)
+  @Column({
+    type: DataType.STRING,
+  })
+    image: string;
```

Comment:
We need to have the option to add not only one image, maybe we can use DataType.ARRAY?